### ISSUE TITLE
Initial treeitem doesn't get read when user lands on / focuses on tree

### ISSUE THREAD

**Issue by akarelas** [nvaccess/nvda#8266]
### Steps to reproduce:
1. Visit this page with NVDA & Firefox: http://s3.karjala.org/tree.html
2. Navigate to the section holding the tree, by pressing 'd' once
3. Press enter to enter Focus mode on the tree.
4. Notice the clicking sound, but also note that the inital treeitem is not read to you

### Expected behavior:
You should hear the contents of the second treeitem (the active one) which is "Super"

### Actual behavior:
You don't hear any words, just the clicking sound

### System configuration:
NVDA version: 2018.1.1

NVDA Installed or portable: installed

Other information: Running in a VM

Windows version: Windows 10 Home most recent version

Name and version of other software in use when reproducing the issue: Firefox 60.0

### Other questions:

Does the issue still occur after restarting your PC? Yes

Have you tried any other versions of NVDA? No

Here's the page code, just in case the HTML file ever changes:

```html
<!DOCTYPE html>
<html lang="en">
<head>
	<title>a page</title>
	<style>
		header {
			padding: 10px;
			font-size: larger;
			background-color: #afa;
		}

		[role=region] {
			margin-top: 20px;
		}

		[role=tree] {
			border: 1px solid black;
			padding: 5px;
		}

		[role=treeitem] {
			border: 1px solid blue;
			padding: 5px;
		}
	</style>
</head>
<body>

<header>My Company Logo</header>

<!-- <input type="text" value="18" /> -->
<!--  aria-activedescendant="t1" -->

<div role="region" aria-label="Forum">
<div role="tree" tabindex="0" aria-activedescendant="t2">
	<div role="treeitem" id="t1" tabindex="-1">Wonderful
		<div role="group">
			<div role="treeitem" id="t2" tabindex="-1">Super</div>
		</div>
	</div>
</div>
</div>

</body>
</html>
```

**Comment by akarelas** [comment]
Should I worry for the fact that I have not received any comments on this bug?

**Comment by akarelas** [comment]
Here is another very good example that remonstrates, where the tree-item programmatically receives the focus when the user presses Enter while on the region that contains the tree:

http://s3.karjala.org/demo-noreport-tree.html

**Comment by ehollig** [comment]
I do not believe Aria role="tree" is fully supported by NVDA as of yet. [Here is another example](https://www.w3.org/WAI/GL/wiki/Using_ARIA_trees).

**Comment by brennanyoung** [comment]
Possibly related: With menus, NVDA fails to read out the accessible name for activedescendant unless the activedescendant is a **direct** descendant of the (focused) element where aria-activedescendant is set. This is not mentioned in the w3c patterns doc, and seems an awkward and unnecessary constraint. It should be possible to wrap your menu in arbitrary divs (e.g. for styling), and as long as the activedescendant points to an document-unique ID, still expect it to work with NVDA so I am about to report this as a separate issue.

**Comment by jcsteh** [comment]
Pressing enter to switch to focus mode never repeats the control; this is expected. (NVDA doesn't do this for any control, tree or otherwise.) However, in Firefox, you do now (after #9166) hear the tree and the active tree view item when you first land on it. This doesn't work in Chrome due to bugs in Chrome which caused problems when we tried to enable this. Once those Chrome bugs are fixed, this will work the same way in Chrome.

**Comment by Adriani90** [comment]
@jcsteh the item is not reported in Firefox 71. IE11 is the only browser which works with this example as expected.
In Chrome and Edge NVDA reports "super not selected" after pressing tab and shift+tab on the tree view. So is the selected status really set correctly from a programatical point of view? Something seems to be wrong here.

**Comment by Adriani90** [comment]
cc: @michaelDCurran

**Comment by Adriani90** [comment]
cc: @MarcoZehe I am not sure if this is bad web authoring here.

**Comment by MarcoZehe** [comment]
I think @jcsteh already explained how this works.

**Comment by Adriani90** [comment]
yes, sorry missed that comment. Closing this issue since the behavior is consistent here.
